dotenv-sample

```diff
@@ -1 +1,5 @@
+# Default environment variables for local development
 
+DJANGO_DEBUG=True
+
+DJANGO_SECRET_KEY="django-insecure-q^d8$ak3811$r^b*@i4tqld6c@5co5in*l=9h@8c!nh4!c$v@l"
```

Comment:
I have a mild preference for something even more obviously insecure here.

Reply:
Fine yes, this was just the default Django generated.